ds3-autogen-java/src/main/resources/tmpls/request/get_object_template.ftl

```diff
@@ -24,6 +24,7 @@ public class ${name} extends AbstractRequest {
 
     // Constructor
 
+<#if helper.containsArgument(optionalArguments, "Job")>
```

Comment:
No. The AmazonS3 needs to print this, but the SpectraS3 does not.  For the SpectraS3, this will generate the same constructor as below, which of course causes a compilation error.  When we refactor to use different generators, this will be much cleaner.  For now, this is the simplest fix.